examples/nextjs/src/app/components/UnauthenticatedView.tsx

```diff
@@ -103,39 +103,40 @@ export default function UnauthenticatedView(): JSX.Element {
       <h3 className="text-lg">Sign in with YouVersion Button</h3>
       <div className="flex flex-wrap gap-4">
         <section className="bg-[rgba(238,238,238,1)] p-4 flex items-center gap-4 w-full">
-          <SignInButton
+          <YouVersionAuthButton
             permissions={[SignInWithYouVersionPermission.bibles]}
             redirectUrl={redirectUrl}
             onAuthError={handleError}
+            mode="signOut"
```

Comment:
Since we're in the unauthenticated view, it seems incorrect to set `mode="signOut"`.